contrib/for-tests/mount-tester/mt.go

```diff
@@ -152,3 +160,41 @@ func readWriteNewFile(path string) error {
 
 	return readFileContent(path)
 }
+
+func readFileContentInLoop(path string) error {
+	if path == "" {
+		return nil
+	}
+	var content []byte
+	//Expected content "mount-tester new file\n", length (22), retry during 240 second
+	content = TestFileContent(path, 22, 240)
```

Comment:
I don't mind but i coud not see why we trouble doing that. 
Our main concern is to set up a proper waiting time, changing it as a parameter won't fix that after all. Giving a comfortable waiting time should do.